exec/java-exec/src/main/java/org/apache/drill/exec/store/easy/json/loader/JsonLoaderImpl.java

```diff
@@ -276,6 +285,10 @@ public boolean readBatch() {
     RowSetLoader rowWriter = rsLoader.writer();
     while (rowWriter.start()) {
       if (parser.next()) {
+        // Add implicit fields
+        if (implicitFields != null) {
```

Comment:
I think the `implicitColumns` may be empty (Not null). So, I recommend that
```java
if(!implicitColumns.isEmpty()) {
  // do that
}
```

Reply:
In the next line it calls the `writeImplicitColumns()` method which has a `foreach` loop.  If the implicit columns are empty (not null) it will skip the loop and return.